marketing/webinars.md

```diff
@@ -1,7 +1,3 @@
-navTitle: Webinars
-navGroup: Sales & Marketing
----
-

```

Comment:
@robmarcer why has this been removed? This is used to categorise the page on the left-side navigation

Reply:
very strange, in the build webinars file it's not removing the breaker to merge the properties:

```
---
originalPath: marketing/webinars.md
updated: 2023-01-31 13:45:55 +0700
---
---
navTitle: Marketing
---
```

but it's only happening with this file...

Reply:
Windows vs Mac/Linux problem...

The file was added with CRLF line endings, so our replace didn't catch it. @knolleary isn't there a way to configure a repo to always ensure consistent line endings?

Reply:
A quick google search suggests so - https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git

In the mean time, lets fix the script to handle the extra `\r`.